R/print.R

```diff
@@ -62,12 +62,20 @@ print.gsplot <- function(x, ...){
     }
 
     print.view(views[[view.name]])
-    par(old.par)
+    
+    
+    if(is.na(as.logical(all.equal(c(1,1), par()$mfrow))) & is.na(as.logical(all.equal(c(1,1,1,1), par()$mfg)))){
```

Comment:
This line is pretty much the meat of this PR. Also where the bug fix will need to happen to get the right log scales in a layout.